assets/js/keyBindings.coffee

```diff
@@ -773,6 +776,12 @@ if module?
           finishes_visual: true
           fn: (cursor, options = {}) ->
             @view.yankBetween @view.cursor, cursor, options
+        CLONE:
+          display: 'Yank blocks as a clone'
+          drop: true # TODO: Is this correct? Re-check after Jeff documents
```

Comment:
yeah drop: true is fine, it means the sequence won't be saved for things like the repeat operator, which i think makes sense here